.github/workflows/unlabel-review.yml

```diff
@@ -0,0 +1,16 @@
+
+name: Reviewed
+# Runs when a core-dev review a PR and remove the "Waiting for Reviewer" label
+on:
+  pull_request_review:
+    types: submitted
+
+jobs:
+  one:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        if: github.event.pull_request.state == 'open'
+        run: |
+          echo "Marking pull request ${{ github.event.pull_request.number }} as reviewed"
+          curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -X "DELETE" '{"labels": ["Waiting for Reviewer"]}' https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/labels
```

Comment:
```suggestion
          curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -X "DELETE" https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/labels/Waiting%20for%20Reviewer
```

Reply:
> > WDYT?

I think that I need to be more focused when I copy paste... ;)